app/views/content/train-to-be-a-teacher/classroom-placements.md

```diff
@@ -0,0 +1,61 @@
+---
```

Comment:
Is it worth adding some links to this page as part of this PR?

Also - do we need to remove some of the placement content on `/train-to-be-a-teacher/initial-teacher-training` and just point to this page instead? (NB there is some additional content on that what to expect page which might be worth adding here if relevant?)